src/app/globals.css

```
@@ -100,3 +100,157 @@ body {
+  }
+
+  /* 코드블록 프린트 대비 */
+  pre,
+  code {
+    white-space: pre-wrap !important;
+    word-break: break-all;
+  }
+
+  /* 페이지 나눔 설정 */
+  h1, h2, h3 {
+    page-break-after: avoid;
```

Stylelint (17.9.0)

[error] Expected "page-break-after" to be "break-after" (property-no-deprecated)

(property-no-deprecated)